.github/workflows/tests.yml

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
This newline is my bad. Sorry - will fix it when merged.

Reply:
i can remove it don't worry :)